README.md

```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+# Amazone Makeup Stock 
+Amazon Makeup project is used for showing list of makeup products to user and also users can filter by brand and price amount. For building this API we have used third party api.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![screenshot](./app_screenshot.png)
 
-## Available Scripts
+## Project Description
+- > [Loom Link](https://www.loom.com/share/f0afd9ca90424c99a64d27a71415b165)
+## Live Demo 
+- > [Heroku Cloud Link](https://az-makeup.herokuapp.com/)
```

Comment:
- [OPTIONAL] The deployed link is not working as expected. I want to recommend that you create some time to fix it.

Reply:
Thanks I am going to fix it